src/Framework/PaymentGateways/Actions/UpdateDonationStatus.php

```diff
@@ -0,0 +1,121 @@
+<?php
+
+namespace Give\Framework\PaymentGateways\Actions;
+
+use Exception;
+use Give\Donations\Models\Donation;
+use Give\Donations\Models\DonationNote;
+use Give\Donations\ValueObjects\DonationStatus;
+use Give\Framework\PaymentGateways\Log\PaymentGatewayLog;
+use Give\Subscriptions\ValueObjects\SubscriptionStatus;
+
+/**
+ * @unreleased
+ */
+class UpdateDonationStatus
```

Comment:
@glaubersilva should we move these actions into the webhooks folder as well? they seem related to those as updating a status can already be done with the models.  It seems these status updates include notes and gateway specific information like transaction ID

Reply:
@jonwaldstein I agree, makes more sense! 

Resolved in 67561161c97f1945ba0fe253e6283e58f2328b61

Reply:
thanks man!